### PR TITLE
en: improve deploy on aws eks

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Improve the format and readability of deploy-on-aws-eks.md

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/802
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -78,74 +76,74 @@ eksctl create cluster -f cluster.yaml
...
-    kubectl create namespace tidb-cluster
-    ```
+> **Note:**
+>
+> A [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) is a virtual cluster backed by the same physical cluster. This document takes `tidb-cluster` as an example. If you want to use other namespace, modify the corresponding arguments of `-n` or `--namespace`.
 
-    > **Note:**
-    >
-    > A [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) is a virtual cluster backed by the same physical cluster. This document takes `tidb-cluster` as an example. If you want to use other namespace, modify the corresponding arguments of `-n` or `--namespace`.
+### Deploy the cluster and its monitoring component
```
Comment (yikeke) [inline en/deploy-on-aws-eks.md:103]:
```suggestion
### Deploy
```
Avoid duplicate headers

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -78,74 +76,74 @@ eksctl create cluster -f cluster.yaml
...
-    {{< copyable "shell-regular" >}}
+{{< copyable "shell-regular" >}}
 
-    ```shell
-    kubectl create -f tidb-cluster.yaml -n tidb-cluster &&
-    kubectl create -f tidb-monitor.yaml -n tidb-cluster
-    ```
+```shell
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/aws/tidb-cluster.yaml -n tidb-cluster && \
+    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/aws/tidb-monitor.yaml -n tidb-cluster
```
Comment (yikeke) [inline en/deploy-on-aws-eks.md:111]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/aws/tidb-monitor.yaml -n tidb-cluster
```
The indent is not necessary?

File: en/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -78,74 +76,74 @@ eksctl create cluster -f cluster.yaml
...
+```
 
-4. View the startup status of the TiDB cluster:
+After the yaml file above is applied to the Kubernetes cluster, TiDB Operator creates the desired TiDB cluster and its monitoring component according to the yaml file.
 
-    {{< copyable "shell-regular" >}}
+### View the cluster starting status
```
Comment (yikeke) [inline en/deploy-on-aws-eks.md:116]:
```suggestion
### View the cluster status
```

File: en/deploy-on-aws-eks.md [hunk 4]
```diff
@@ -78,74 +76,74 @@ eksctl create cluster -f cluster.yaml
...
 
-    ```shell
-    kubectl get pods -n tidb-cluster
-    ```
+To view the starting status of the TiDB cluster, run the following command:
```
Comment (yikeke) [inline en/deploy-on-aws-eks.md:118]:
```suggestion
To view the status of the starting TiDB cluster, run the following command:
```

File: en/deploy-on-aws-eks.md [hunk 5]
```diff
@@ -218,14 +217,12 @@ After the bastion host is created, you can connect to the bastion host via SSH a
     6 rows in set (0.00 sec)
     ```
 
-    `<tidb-nlb-dnsname>` is the LoadBalancer domain name of the TiDB service. You can view the domain name in the `EXTERNAL-IP` field by executing `kubectl get svc basic-tidb -n tidb-cluster`.
-
 > **Note:**
 >
-> * [The default authentication plugin of MySQL 8.0](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin) is updated from `mysql_native_password` to `caching_sha2_password`. Therefore, if you use MySQL client from MySQL 8.0 to access the TiDB service (TiDB version < v4.0.7), and if the user account has a password, you need to explicitly specify the `--default-auth=mysql_native_password` parameter.
+> * [The default authentication plugin of MySQL 8.0](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin) is updated from `mysql_native_password` to `caching_sha2_password`. Therefore, if you use MySQL client from MySQL 8.0 to access the TiDB service (< v4.0.7), and if the user account has a password, you need to explicitly specify the `--default-auth=mysql_native_password` parameter.
```
Comment (yikeke) [inline en/deploy-on-aws-eks.md:222]:
```suggestion
> * [The default authentication plugin of MySQL 8.0](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin) is updated from `mysql_native_password` to `caching_sha2_password`. Therefore, if you use MySQL client from MySQL 8.0 to access the TiDB service (cluster version < v4.0.7), and if the user account has a password, you need to explicitly specify the `--default-auth=mysql_native_password` parameter.
```
Suggest keeping it to make it clearer

File: en/deploy-on-aws-eks.md [hunk 6]
```diff
@@ -348,7 +358,7 @@ For detailed CR configuration, refer to [API references](https://github.com/ping
 
 ## Deploy TiDB Enterprise Edition
 
-If you need to deploy TiDB/PD/TiKV/TiFlash/TiCDC Enterprise Edition, configure `spec.<tidb/pd/tikv/tiflash/ticdc>.baseImage` in `tidb-cluster.yaml` as the enterprise image. The image format is `pingcap/<tidb/pd/tikv/tiflash/ticdc>-enterprise`.
+To deploy TiDB/PD/TiKV/TiFlash/TiCDC Enterprise Edition, configure `spec.[tidb|pd|tikv|tiflash|ticdc].baseImage` in `tidb-cluster.yaml` as the enterprise image. The Enterprise image format is `pingcap/[tidb|pd|tikv|tiflash|ticdc]-enterprise`.
```
Comment (yikeke) [inline en/deploy-on-aws-eks.md:361]:
```suggestion
To deploy TiDB/PD/TiKV/TiFlash/TiCDC Enterprise Edition, configure `spec.[tidb|pd|tikv|tiflash|ticdc].baseImage` in `tidb-cluster.yaml` as the enterprise image. The enterprise image format is `pingcap/[tidb|pd|tikv|tiflash|ticdc]-enterprise`.
```

File: en/deploy-on-aws-eks.md [hunk 7]
```diff
@@ -262,25 +258,37 @@ The upgrade process does not finish immediately. You can watch the upgrade progr
...
+After scaling out the EKS node group, execute `kubectl edit tc basic -n tidb-cluster`, and modify each component's `replicas` to the desired number of replicas. The scaling-out process is then completed.
+
 ## Deploy TiFlash/TiCDC
 
+[TiFlash](https://docs.pingcap.com/tidb/stable/tiflash-overview) is the columnar storage extension of TiKV.
+
+[TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview) is a tool for replicating the incremental data of TiDB by pulling TiKV change logs.
+
+The two components are *not required* in the deployment. This section shows a quick start example.
+
```
Comment (dragonly) [inline en/deploy-on-aws-eks.md:286]: We can change the Chinese version to the same 3-paragraph layout here

**Comment by dragonly** [comment]
rest LGTM

File: en/deploy-on-aws-eks.md (hunk 7 above)
Comment (ran-huang) [reply en/deploy-on-aws-eks.md:286]: Updated.

**Comment by ran-huang** [comment]
@yikeke PTAL again. Thanks!

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**